src/main/java/com/streamingplatform/moviestreamingplatform/model/dto/mapper/CustomMapper.java

```diff
@@ -11,12 +11,10 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import javax.annotation.processing.Generated;
-
 import org.springframework.stereotype.Component;
 
 @Component
-public class myMapper implements ImyMapper {
+public class CustomMapper implements ICustomMapper {
```

Comment:
@Ruleabogdan try replacing this class with custom constructors for each mapping. Also, when you want to transform a list into another list try using "stream" and "map" to have a functional programming style